code/__DEFINES/genetics.dm

```diff
@@ -27,6 +27,7 @@
 
 // Extra powers:
 #define LASER			9 	// harm intent - click anywhere to shoot lasers from eyes
+#define GREY			10	//Wingdings for Greys -- Not activatable or researchable
```

Comment:
This being a mutation is.. rather iffy.
I know no systems really exist for species font classes, but one probably should.